build.sbt

```diff
@@ -162,7 +162,8 @@ lazy val query = (project in file("query"))
         oldStrategy(x)
     },
     parallelExecution in Test := false,
-    libraryDependencies ++= configDeps ++ akkaPersistenceDeps ++ slickDeps
+    fork in Test := true,
```

Comment:
It looks like this line is causing the `java.io.EOFException` that causes the "sbt.ForkMain failed with exit code 137" error in Travis.

Is forking necessary in the `query` project's specs? If so, why?